HeterogeneousCore/SonicTriton/scripts/cmsTriton

```diff
@@ -38,17 +40,18 @@ usage() {
 	$ECHO "-c          \t don't cleanup temporary dir (for debugging)"
 	$ECHO "-C [dir]    \t directory containing Nvidia compatibility drivers (checks CMSSW_BASE by default if available)"
 	$ECHO "-D          \t dry run: print container commands rather than executing them"
-	$ECHO "-d          \t use Docker instead of Apptainer"
+	$ECHO "-d          \t use Docker instead of Singularity"
```

Comment:
please fix the rebase, every change from Apptainer back to Singularity is a regression.

Reply:
Yes, sorry I copied that in from an older repository.  Went ahead and edited the new version by hand now